src/components/Lookup/index.js

```diff
@@ -352,44 +352,62 @@ class Lookup extends Component {
 }
 
 Lookup.propTypes = {
+    /** Text label for the Lookup. */
+    label: PropTypes.oneOfType([
+        PropTypes.string, PropTypes.node,
+    ]).isRequired,
+    /** A boolean to hide the Lookup label. */
+    hideLabel: PropTypes.bool,
     value: PropTypes.shape({
         label: PropTypes.string,
         description: PropTypes.string,
         icon: PropTypes.node,
     }),
+    options: PropTypes.arrayOf(
+        PropTypes.shape({
+            label: PropTypes.string,
+            description: PropTypes.string,
+            icon: PropTypes.node,
+        }),
+    ),
+    /** The name of the Lookup. */
     name: PropTypes.string,
-    label: PropTypes.oneOfType([
-        PropTypes.string, PropTypes.node,
-    ]).isRequired,
+    debounce: PropTypes.bool,
+    /** If is set to true, then is showed a loading symbol. */
+    isLoading: PropTypes.bool,
+    /** Text that is displayed when the field is empty, to prompt the user for a valid entry. */
     placeholder: PropTypes.string,
+    /** Specifies that the Lookup must be filled out before submitting the form.
+    * This value defaults to false. */
     required: PropTypes.bool,
+    /** Specifies that the Lookup must be filled out before submitting the form. */
     error: PropTypes.oneOfType([
         PropTypes.string, PropTypes.node,
     ]),
+    /** Specifies that the Lookup element should be disabled. This value defaults to false. */
     disabled: PropTypes.bool,
+    /** Specifies that the Lookup is read-only. This value defaults to false. */
     readOnly: PropTypes.bool,
-    onChange: PropTypes.func,
+    /** Specifies the tab order of an element (when the tab button is used for navigating). */
     tabIndex: PropTypes.oneOfType([
         PropTypes.number,
         PropTypes.string,
     ]),
+    onSearch: PropTypes.func,
```

Comment:
The action triggered for every key stroke when the customer is typing in the input. It sent the value/query of the input. This value is normally used for filter/search for more options.